tests/test_definitions.py

```diff
@@ -32,14 +32,14 @@ def test_variables_industry_types():
     assert obs["description"] == exp
 
 
-def test_variables_transport_types():
-    # check that exploding of {transport} to transportation modes works
-    obs = definition.variable["Energy Service|Transportation|Freight|Rail"]
-    exp = (
-        "Provision of energy services related to freight "
-        "rail-based transportation technologies"
-    )
-    assert obs["description"] == exp
+#def test_variables_transport_types():
```

Comment:
Commenting out a test is only a good idea if you intend to re-activate it later. It is not clear to what this PR changed to make the test fail...

Reply:
As I mentioned above, Freight|Rail is not handled anymore by the {Transport} tag. To avoid a duplicated variable name error, the entry had to be removed from the {Transport} tag, which in turn caused this test to error out by default.

I commented it out only because I had no idea how you would like to proceed about that. 

The {Transport} tag became a legacy tag, and it was only left in the code due to the requirement to keep supporting non NAVIGATE variables.
In order to represent the NAVIGATE template variable it was necessary to redo the transport tag system in order support the new structure in transportation modes, technologies and carriers.

Reply:
Rail is not handled anymore by the {Transport} tag. To avoid a duplicated variable name error, the entry had to be removed from the {Transport} tag, which in turn caused this test to error out by default.

I commented it out only because I had no idea how you would like to proceed about the test: either remove it (1), or adapt it to the new NAVIGATE tags (tag_transport_mode.yaml, tag_transport_mode_and_carrier.yaml and tag_transport_mode_tech_and_carrier.yaml).

The {Transport} tag became a legacy tag, and it was only left in the code due to the requirement to keep supporting non NAVIGATE variables.
In order to represent the NAVIGATE template variable it was necessary to redo the transport tag system to support the new structure in transportation modes, technologies and carriers.

Reply:
Coming is here as Daniel is on holiday, please excuse me as I might get this wrong.
As far as I read the test, it does not actually matter what the tag is called in the end `{Transport}` or in the newer case `{Transport Mode}`.
From what I gather, the test fails because you slightly changed the description from "rail-based transportation technologies" to "rail-based transportation". As the test is for an exact match, it fails.
You should be able to fix it by either changing `exp` the test to end with "rail-based transportation" or changing the description of `Rail` to "rail-based transportation technologies". 
You should also then be able to remove the former definition of `Rail` from `tag_transport_types.yaml`.

Reply:
I am also not sure what was the intention behind the initial test and I did not dig deeper on checking its inner workings. 
I must say that after 5 years not using it, my python skills are a bit rust :) 

I simply commented it because it fails every time, so just let me know if I should simply delete the test code in the pull request, or if one of you could do a code commit suggestion on what should be changed to adapt the test to the new situation.

Reply:
handled in commit (https://github.com/openENTRANCE/openentrance/pull/165/commits/d249541f68daf901c990a93be901c34767f434cf)